docs/content/how-to/creating-catalogs.md

```diff
@@ -163,3 +163,16 @@ You can set location in the properties of table/database by the config of `locat
 By default, Paimon does not synchronize newly created partitions into Hive metastore. Users will see an unpartitioned table in Hive. Partition push-down will be carried out by filter push-down instead.
 
 If you want to see a partitioned table in Hive and also synchronize newly created partitions into Hive metastore, please set the table property `metastore.partitioned-table` to true. Also see [CoreOptions]({{< ref "maintenance/configurations#CoreOptions" >}}).
+
+### Adding Parameters to a Hive Table
+
+Using the `--table-conf` parameter allows for the convenient definition of Hive table parameters. When you utilize configurations
+that start with “hive.”, these configurations will be defined in the `TBLPROPERTIES` of the Hive Table.
+for example:
+```bash
```

Comment:
remove this example, you can just say "if you add table option `hive.table.owner`, then ....."

Reply:
done.